x/ccv/consumer/keeper/changeover.go

```diff
@@ -0,0 +1,34 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func (k Keeper) ChangeoverToConsumer(ctx sdk.Context) (initialValUpdates []abci.ValidatorUpdate) {
+
+	initialValUpdates = k.GetInitialValSet(ctx)
+	// set last sovereign height
+	k.SetLastSovereignHeight(ctx, ctx.BlockHeight())
+	// populate cross chain validators states with initial valset
+	k.ApplyCCValidatorChanges(ctx, initialValUpdates)
+
+	// Add validator updates to initialValUpdates, such that the "old" validators returned from sovereign staking module
```

Comment:
Maybe we should remove any instance of sovereign as this would imply that the consumer is not sovereign, which is false. Please use phrases like `existing chain`, `previous staking module` ...

Reply:
I agree that the naming here is difficult. But a consumer chain is definitely less sovereign than a normal cosmos chain that is responsible for it's own valset. Using a term like `existing chain` doesn't fully get the point across imo